src/frontend/app/app-public/app-public.tsx

```diff
@@ -20,7 +20,7 @@ const AppPublic: React.FC = () => {
   document.body.style.backgroundColor = theme.bgColor;
 
   return (
-    <html className={theme.name}>
+    <html className={theme.cardBg}>
```

Comment:
this is not always true in the case of expanded themes -- the name field of the theme is meant to be the class name

Reply:
we'll need some kind of translation from a given theme to a unique identifier. what do you suggest?

Reply:
and then className={theme.className}